model/src/main/java/com/flipkart/flux/domain/StateMachine.java

```diff
@@ -158,6 +166,14 @@ public Timestamp getUpdatedAt() {
         return updatedAt;
     }
 
+    public String getClientElbId() {
+        return clientElbId;
+    }
+
+    public void setClientElbId(String clientElbId) {
```

Comment:
no setter, to be initialized one time in constructor

Reply:
Removed setter for ClientElbId.